lib/ansible/modules/windows/win_package.ps1

```diff
@@ -1259,23 +1259,58 @@ $result = @{
     changed = $false
 }
 
-$path = Get-Attr -obj $params -name path -failifempty $true -resultobj $result
-$name = Get-Attr -obj $params -name name -default $path
-$productid = Get-Attr -obj $params -name productid
-if ($productid -eq $null)
+$path = Get-AnsibleParam -obj $params -name "path" -failifempty $true -resultobj $result
+$name = Get-AnsibleParam -obj $params -name "name" -default $path
+$productid = Get-AnsibleParam -obj $params -name "productid" -aliases "product_id"
+$arguments = Get-AnsibleParam -obj $params -name "arguments"
```

Comment:
Can we set a deprecation warning if `arguments` is set to tell the users to use `options` intead? Will need to add it to the docs if this is the case.